<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>103</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>设置男生年龄</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxBoy">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>147</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBoyInc">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>boy长大一岁</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClassInfo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>类的元对象信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>108</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>设置女生年龄</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxGirl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>147</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGirlInc">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>girl长大一岁</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>清空文本框</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
